examples/process-compose/devenv.nix

```diff
@@ -7,16 +7,28 @@
 
   services.postgres.enable = true;
 
+  languages.ruby.enable = true;
+
+  packages = [ pkgs.imagemagick6 ];
+
+  scripts.compile-rmagick.exec = "gem install --install-dir /tmp rmagick";
```

Comment:
It would be better to move this into https://github.com/cachix/devenv/blob/main/examples/ruby/devenv.nix

Reply:
I don't want to deprecate the `createDatabase` flag just yet, to keep the amount of breaking changes minimal.
Could the `compile-rmagick` line stay here? It tests that the `scripts` are in the `PATH` of `process-compose`. Hitting 2 birds with 1 stone.

Reply:
:+1: